addons/web/static/src/legacy/scss/daterangepicker.scss

```diff
@@ -1,69 +1,119 @@
-                    color: #FFFFFF;
-                    background-color: $o-brand-odoo;
-                    th {
-                        &.prev, &.next {
-                            span {
-                                color: #FFFFFF;
-                                border-color: #FFFFFF;
-                            }
-                            &:hover {
-                                background-color: darken($o-brand-odoo, 10%);
-                            }
-                        }
-                    }
-                }
-                tr:last-child {
-                    color: $o-datepicker-week-color;
-                    background-color: $o-datepicker-week-bg-color;
-                }
-                th {
-                    border-radius: 0;
-                }
+    font-family: inherit;
+
+    .calendar-table {
+        td, th {
+            border-bottom-color: $table-border-color;
+
+            &, &.start-date.end-date {
+                border-radius: 0;
+            }
+
+            &:not(.active) {
+                color: gray('900');
+            }
+
+            &.prev, &.next, &.off.ends {
+                @include o-hover-opacity(.5);
             }
-            tbody {
-                tr {
-                    border-bottom: 1px solid $o-datepicker-week-bg-color;
-                    td {
-                        border-width: 0px;
-                        &:not(.off) {
-                            &.in-range {
-                                background-color: gray('200');
-                            }
-                            &.available:hover {
-                                background-color: gray('200');
-                            }
-                            &.active, &.active:hover {
-                                background-color: $o-brand-primary;
-                            }
-                        }
-                        &.off:hover {
-                            background: #FFFFFF;
-                            color: #999;
-                        }
-                    }
+        }
+
+        td {
+            font-size: ($font-size-base + $font-size-sm) * .5;
+
+            &.today {
+                position: relative;
+
+                &:before {
+                    content: '';
+                    display: inline-block;
+                    border: solid transparent;
+                    border-width: 0 0 7px 7px;
+                    border-bottom-color: $o-brand-primary;
+                    border-top-color: $dropdown-border-color;
+                    position: absolute;
+                    bottom: 4px;
+                    right: 4px;
                 }
             }
+
+            &.active,
+            &.active:hover,
+            &.in-range {
+                color: $white;
+                text-shadow: 0 1px 0 rgba(0, 0, 0, 0.25);
+            }
+
+            &.in-range {
+                background-color: lighten($o-brand-odoo, 40%);
```

Comment:
I find the "in-range" color way too pink, even for odoo ^^
More importantly these colors fails WCAG tests.
![Screenshot 2021-09-20 at 14 32 11](https://user-images.githubusercontent.com/8558351/134006764-915eb79e-be41-4d17-a425-6386d58014fe.png)

---

I'd suggest to use the `rgba` function rather than the `lighten` one.
These values should work fine (tested in the browser, not in the actual code)

```scss
&.in-range {
    background-color: rgba($o-brand-odoo, 0.3);
    color: $o-main-text-color;
    text-shadow: none;
}
```


![Screenshot 2021-09-20 at 14 53 18](https://user-images.githubusercontent.com/8558351/134007058-669102ab-a763-4c1d-a968-a98cb0320cb3.png)

Reply:
Nice catch! :wink: 
Your proposition has been applied, it should be good now.